### PR TITLE
Release v0.6.0

### PR DESCRIPTION
**Pull request by jusjusjus**

### CODE REVIEW

File: mffpy/raw_bin_files.py [hunk 1]
```diff
@@ -34,45 +34,70 @@ def __init__(self, filepointer: IO[bytes]):
         assert not self.filepointer.closed
         self.buffering: bool = False
 
-    def __del__(self):
-        self.close()
+    def read_raw_samples(self, t0: float = 0.0,
```
Comment (jusjusjus) [inline mffpy/raw_bin_files.py:37]: This function was moved further to the top; nothing deleted nothing added.

File: mffpy/xml_files.py [hunk 2]
```diff
@@ -153,46 +154,85 @@ class FileInfo(XML):
...
     @classmethod
     def content(cls, recordTime: datetime,  # type: ignore
-                mffVersion: str = '3') -> dict:
-        """returns mffVersion and time of recording start
+                mffVersion: str = '3',
+                acquisitionVersion: str = None,
+                ampType: str = None) -> dict:
+        """returns MFF file information
 
         As version we only provide '3' at this time.  The time has to provided
```
Comment (ephathaway) [inline mffpy/xml_files.py:192]:
```suggestion
        For version we only provide '3' at this time.  The time has to be provided
```

**Review by ephathaway — APPROVED** [review]
Looks good. Just a few documentation things.

As far as the version number, is this a special release or does 0.6.0 follow 0.5.9 by convention (as opposed to 0.5.10)?

**Comment by GeorgeHernandez541** [comment]
Agreed. Should be good to merge after Evan's suggestions are addressed.

**Review by ephathaway — APPROVED** [review]

**Pull request merged**